File: test/fixtures/output/csharp/httpclient/text-plain.cs
using System;
using System.Net.Http;
using System.Text;
using System.Threading.Tasks;

class Snippet
{
    static HttpClient client = new HttpClient();

    static async Task MainAsync()
    {
        var request = new HttpRequestMessage(HttpMethod.Post, "http://mockbin.com/har");
        request.Content = new StringContent("Hello World",
            Encoding.UTF8, "text/plain");

        var response = await client.SendAsync(request);
        var content = await response.Content.ReadAsStringAsync();
        Console.WriteLine(content);
    }

    static void Main(string[] args)
    {
        MainAsync().Wait();
    }
}

File: __tests__/__fixtures__/output/php/guzzle/text-plain.php
<?php

$client = new \GuzzleHttp\Client();

$response = $client->request('POST', 'https://httpbin.org/anything', [
  'body' => 'Hello World',
  'headers' => [
    'content-type' => 'text/plain',
  ],
]);

echo $response->getBody();
